common/config/rush/command-line.json

```diff
@@ -180,7 +228,15 @@
 
       // For non-phased commands, we will continue to use the "associatedCommands" setting to associate
       // parameters with them.  (Phased commands must NOT appear in this list.)
-      "associatedCommands": ["build"]
+      "associatedCommands": ["build"],

```

Comment:
If this is a realistic example, we should show the intended Rush shell commands for realistic everyday tasks:

- How do I build everything including all phases?
- How do I update the README and do nothing else?
- How do I do a `--lite` build where we skip unit tests and linting?
- For these things, what's the equivalent of `rush rebuild`?

Maybe the answer is that initially each of these problems requires a completely new command action name.  (`rush build-lite`?)  But if so we can still enumerate them to get a sense of how painful that feature cut will be in practice.